### ISSUE TITLE
Fix Link to Print Spooler Blogpost in Disabling System Services Security Guidelines

### ISSUE THREAD

**Issue by no-identd** [MicrosoftDocs/windowsserverdocs#5670]
The link in the print spooler section should point to https://techcommunity.microsoft.com/t5/ask-the-performance-team/disabling-unnecessary-services-a-word-to-the-wise/ba-p/373444, but the automatic website porting stuff you guys have messed this—once again—up, so I had to figure this out via archive.org. Please fix. (I might even send a pull request if I get around to it…) #PrintNightmare

(btw.: the original author of that blog post wasn't Craig Marcho as the current page claims, the original author is the late Suhail "CC" Hameed, who unfortunately passed on in 2011.)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7d867f27-5abb-be93-6bad-13eaa2e57ca6
* Version Independent ID: 8e8d603f-f93f-57f2-dec2-098433885624
* Content: [Security guidelines for system services in Windows Server 2016](https://docs.microsoft.com/en-us/windows-server/security/windows-services/security-guidelines-for-disabling-system-services-in-windows-server#print-spooler)
* Content Source: [WindowsServerDocs/security/windows-services/security-guidelines-for-disabling-system-services-in-windows-server.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/security/windows-services/security-guidelines-for-disabling-system-services-in-windows-server.md)
* Product: **windows-server**
* Technology: **security**
* GitHub Login: @nirb
* Microsoft Alias: **nirb**

**Comment by ocdtrekkie** [comment]
I apparently duplicated your PR a bit, though I selected a different blog post in mine: https://techcommunity.microsoft.com/t5/ask-the-performance-team/printing-and-active-directory/ba-p/373900

I'm a bit ambivalent on which is better. Both explain the use of the Print Spooler service on domain controllers, mine is a more technical entry, but yours is more targeted. Either way, it'd be really nice if these PRs weren't all blocked on a user who doesn't appear to be actively contributing to public GitHub repos.

**Comment by no-identd** [comment]
@sethmanheim @huypub @femila @ocdtrekkie thanks y'all! I wish this kind of think would get fixed via automation however, there's migration issues like this littered all over the place and it's annoying, but I lack high hopes for that…

P.S.:
@sethmanheim, you might want to have the do-not-merge label/tag removed from #5671 but—obviously—not #5703, since it's obviously inapplicable. Don't know what kind of internal reporting you guys have & whether, but it might mess that up otherwise?

**Comment by sethmanheim** [comment]
That is a good idea, Mr/Ms No Identity. I have done that. Thanks!

**Comment by no-identd** [comment]
Actually, maybe I was a bit premature there, as I just found an issue with your merge @sethmanheim, it unintentionally breaks this commit:

https://github.com/MicrosoftDocs/windowsserverdocs/commit/91efb80ca9af2a6ba0ec346ceca3dcc5e37b597d

by @v-lianna

**Comment by no-identd** [comment]
Ideally, you'd have both links in there? Or maybe just all three at this point 🤷🏼‍♂️

**Comment by sethmanheim** [comment]
> Actually, maybe I was a bit premature there, as I just found an issue with your merge @sethmanheim, it unintentionally breaks this commit:
> 
> [91efb80](https://github.com/MicrosoftDocs/windowsserverdocs/commit/91efb80ca9af2a6ba0ec346ceca3dcc5e37b597d)
> 
> by @v-lianna

Can you define "breaks this commit?" I am seeing that commit in the source Markdown in the private repo, so it should be there. Or do you mean the content is now wrong?

**Comment by no-identd** [comment]
@sethmanheim the latter (albeit I wouldn't say 'wrong', just, counter-to-intentions and incomplete), I could have phrased that better however. My pull request fixed a broken link. When @v-lianna made her commit, she (I infer) intended to append a pointer to https://docs.microsoft.com/en-us/defender-for-identity/cas-isp-print-spooler in the form a relative hyperlink, and when preparing to do so, encountered—because my pull request wasn't yet merged—the broken link my pull request intended to fix, was unaware of the (rather obscure) possibility of fixing the broken link by pointing it to where it should have pointed all along (at least if we go by the intent of whoever originally added the link ages ago, I'd guess it was Suhail "CC" Hameed, seeing how he wrote the blog post, but who knows), and thus—instead of appending her presumably originally intended as additional, not substituary, content—opted to replace the line containing the broken link with her content. After that had happened, you merged my original pull request, which then overwrote her content.

I hope that cleared up what I meant now?

P.S.:
At this point, I feel it mandatory to point to http://justinhileman.info/article/git-pretty/ :p

P.P.S.:
Regarding your earlier comment, _actually_ , I just really hate https://manpages.debian.org/jessie/pidentd/identd.8.en.html ;)

**Comment by sethmanheim** [comment]
OK, thanks for the clarification. I have been trying to clean up the backlog of PRs but I am not a regular writer on Windows Server. So I am sorry that I overwrote the PR. So you're saying that the link at the top of this PR is the correct one?

**Comment by no-identd** [comment]
…kind of. I'll try to clarify tomorrow, if @v-lianna hasn't done so herself by then or sent a corrective new PR. But basically, that section of the article, at least the way I view it, should contain, separately:

- a pointer to https://techcommunity.microsoft.com/t5/ask-the-performance-team/disabling-unnecessary-services-a-word-to-the-wise/ba-p/373444 in the form of an absolute path hyperlink (why? Documentation Bugfix.)
- A pointer to  https://docs.microsoft.com/en-us/defender-for-identity/cas-isp-print-spooler in the form of a relative hyperlink (why? idqk, depends on what drove @v-lianna, I'd guess some MS-internal requirement/request from either the MSRC, the Defender for Identity, and/or the Cloud App Security teams? idk)

And since we're at a point of overengineering it by now, it might make sense to additionally include the link @ocdtrekkie offered. idk
